code/modules/mob/living/silicon/pai/pai.dm

```diff
@@ -77,7 +77,7 @@
 	var/radio_short_cooldown = 3 MINUTES
 	var/radio_short_timerid
 
-	canmove = FALSE
+	mobility_flags = NONE
```

Comment:
it's NONE here...

Reply:
pAIs not in card form are supposed to have ZERO interaction with the world other than their programs so yes.